docs/standard/collections/selecting-a-collection-class.md

```diff
@@ -14,59 +14,63 @@ author: "mairaw"
 ms.author: "mairaw"
 ---
 # Selecting a Collection Class
-Be sure to choose your collection class carefully. Using the wrong type can restrict your use of the collection. In general, avoid using the types in the <xref:System.Collections> namespace unless you are specifically targeting .NET Framework version 1.1. The generic and concurrent versions of the collections are to be preferred because of their greater type safety and other improvements.  
-  
+
+Be sure to choose your collection class carefully. Using the wrong type can restrict your use of the collection.  
+
+> [!IMPORTANT]
+> Avoid using the types in the <xref:System.Collections> namespace unless you are specifically targeting .NET Framework 1.1. The generic and concurrent versions of the collections are recommended because of their greater type safety and other improvements.  
```

Comment:
Do you think we still need to talk about 1.1 here?

Reply:
Good point, @mairaw. I think we can eliminate mention of 1.1.